README.md

```diff
@@ -1,5 +1,7 @@
 # Foteli
 
+<p align="center"><img src="https://github.com/google/foteli/blob/main/doc/foteli.svg" alt="Foteli" width="64"></p>
```

Comment:
Out of curiosity, any reason not to make this a relative path?

```suggestion
<p align="center"><img src="doc/foteli.svg" alt="Foteli" width="64"></p>
```

Reply:
Not sure, we always do so. I believe it is because `README` file can be embedded (not in iframe), e.g. on title page of repo.

Reply:
I see, thanks. As far as I know, GitHub takes that into account when processing the Markdown. Let’s try with relative first and switch to absolute if there are any issues?

Reply:
Sure, why not. Will update soon

Reply:
Done. Even preview works